pwa-devdocs/src/tutorials/cloud-deploy/index.md

```diff
@@ -11,51 +11,14 @@ title: Magento Cloud deployment
     You are not required to publish to npmjs.com, but
     NPM or Yarn should be able to access your project code through Git.
 
-For this tutorial, the [`@magento/venia-concept`][] package for the [Venia storefront][] is used, but any PWA available though NPM with an UPWARD compatible YAML file is supported.
+For this tutorial, the [`@magento/venia-concept`][] package for the [Venia storefront][] is used as a template, but any PWA available though NPM with an UPWARD compatible YAML file is supported.
 
-## Add node package dependency
-
-If it does not exist, [create a `package.json`][] file in your project directory.
-
-```sh
-npm init -y
-```
-
-Install your PWA Studio storefront:
-
-```sh
-npm install @magento/venia-concept
-```
-
-This adds the following dependency to the `package.json` file:
-
-```json
-{
-    "dependencies": {
-        "@magento/venia-concept": "~4.0.0"
-    }
-}
-```
+These instructions provide a method for building your application bundle in the Magento Cloud, but
+you can get the same results by building locally and checking your local build artifacts into source control.
```

Comment:
```suggestion
you can get the same results by building locally with the same environment variables as your targeted production environment, and then checking your local build artifacts into source control.
```

Reply:
My edit could maybe be phrased better, but I think this is important to note, since environment variables could cause the remote build and the local build to produce different artifacts.

You'd hope they would not, because the local simulated "production" environment should include all the non-secret production variables, you don't want to hardcode any secrets into the bundle.

Reply:
To my knowledge, the only runtime variables are for UPWARD, so just `NODE_ENV` and `MAGENTO_BACKEND_URL`, all others are baked in. This includes `CHECKOUT_BRAINTREE_TOKEN`, but I always considered that a public key that wasn't useful by itself. Definitely worth auditing if that's not the expectation.